sdk/keyvault/azure-security-keyvault-keys/src/key_client.cpp

```diff
@@ -202,6 +202,28 @@ Azure::Security::KeyVault::Keys::DeleteKeyOperation KeyClient::StartDeleteKey(
           {_detail::KeysPath, name}));
 }
 
+Azure::Security::KeyVault::Keys::DeleteKeyOperation KeyClient::ResumeDeleteKey(
+    std::string const& resumeToken,
+    Azure::Core::Context const& context) const
+{
+  Azure::Security::KeyVault::Keys::DeleteKeyOperation operation(m_pipeline, resumeToken);
+  // Need to fix this to use context directly. See:
+  // https://github.com/Azure/azure-sdk-for-cpp/issues/2091
+  operation.Poll(context.GetApplicationContext());
```

Comment:
This is not the right context to use; use the parameter

Reply:
Added a comment about why I can't use the context parameter. The parameter is a const& and the Poll method takes a non-const context.  I added the gh issue as a comment there